### ISSUE TITLE
 Is the label predicted by the model consistent with the label of the dataset? 

### ISSUE THREAD

**Issue by coderSkyChen** [kenshohara/3D-ResNets-PyTorch#100]
In UCF101,for example, the model predicts 0 for a video, does it means ApplyEyeMakeup? Or, do you have another dict which maps the output of the model to another?

**Comment by coderSkyChen** [comment]
![default](https://user-images.githubusercontent.com/16058650/45631993-ddae0a00-bace-11e8-82a5-0e9b9e85c5a9.png)
![default](https://user-images.githubusercontent.com/16058650/45632012-e6064500-bace-11e8-922b-29ad222c90da.png)
In the picture the first column is the  result predicted by your pretrained model, and the second is the ground truth, so, it seems like that 3 corresponds to 43，and 76 corresponds to 50 ???
I'm very confused that i do not find any dict that maps the output of a model to its ground truth, it is so odd. By the way, i use python2
